app/controllers/admin/invitations_controller.rb

```diff
@@ -27,18 +27,28 @@ def create
                    username: username,
                    remote_profile_image_url: ::Users::ProfileImageGenerator.call,
                    registered: false)
-      flash[:success] = "The invite has been sent to the user's email."
+      flash[:success] = t("admin.invitations_controller.create_success")
       redirect_to admin_invitations_path
     end
 
     def destroy
       @invitation = User.where(registered: false).find(params[:id])
       if @invitation.destroy
-        flash[:success] = "The invitation has been deleted."
+        flash[:success] = t("admin.invitations_controller.destroy_success", email: @invitation.email)
```

Comment:
I put these messages in the locale files because otherwise the pre commit hook wasn't happy with me 😅 

There is some change to the confirmation message copy (namely that it now includes the affected user's email) - this is going to affect the current invitations view as well as the one behind the feature flag. I figured this is OK though, since it represents an improvement to the copy and not any change in behaviour.